sdcm/tester.py

```diff
@@ -3115,6 +3115,7 @@ def wait_data_dir_reaching(self, size, node):
     def check_latency_during_ops(self, op_is_upgrade=False):
         start_time = self.start_time if not self.create_stats else self._stats["test_details"]["start_time"]
         end_time = time.time()
+        nemesis_class_name = self.params.get('nemesis_class_name') or 'upgrades'
```

Comment:
agree with @fruch that https://github.com/scylladb/scylla-cluster-tests/pull/6679 is probably a better solution because this line brings "magic logic" where we assume that if it is not a nemesis test it is an upgrades test

For me, it looks the same as
1) sct.py::send_email logic depends on JOB_NAME. 
2) tricky logic around ClusterTester.get_email_data for proper work of making  email in PerformanceRegressionUpgradeTest we have to force it to return {}

```
class PerformanceRegressionUpgradeTest(PerformanceRegressionTest, UpgradeTest):  # pylint: disable=too-many-ancestors
    def get_email_data(self):  # pylint: disable=no-self-use
        return PerformanceRegressionTest.get_email_data(self)
```

all these 3 solutions make our life harder